src/components/DayPickerNavigation.jsx

```diff
@@ -151,19 +157,21 @@ function DayPickerNavigation({
         hasInlineStyles && inlineStyles,
       )}
     >
-      {!isVerticalScrollable && (
-        renderNavPrevButton ? renderNavPrevButton({
-          ariaLabel: phrases.jumpToPrevMonth,
-          disabled: disablePrev,
-          onClick: disablePrev ? undefined : onPrevMonthClick,
-          onKeyUp: disablePrev ? undefined : (e) => {
-            const { key } = e;
-            if (key === 'Enter' || key === ' ') onPrevMonthClick(e);
-          },
-          onMouseUp: disablePrev ? undefined : (e) => {
-            e.currentTarget.blur();
-          },
-        }) : (
+      {showNavPrevButton &&
+        (renderNavPrevButton ? (
+          renderNavPrevButton({
+            ariaLabel: phrases.jumpToPrevMonth,
+            disabled: disablePrev,
+            onClick: disablePrev ? undefined : onPrevMonthClick,
+            onKeyUp: disablePrev ? undefined : (e) => {
```

Comment:
```suggestion
            onKeyUp: disablePrev ? undefined : ({ key }) => {
```

Reply:
i don't think i can do this since i need to pass e into `onPrevMonthClick`, and same for `onNextMonthClick` below

Reply:
ah, good point, my bad